src/commands.ts

```diff
@@ -34,7 +34,8 @@ function ensurePR(prManager: IPullRequestManager, pr?: PRNode | IPullRequestMode
 	}
 }
 
-export function registerCommands(context: vscode.ExtensionContext, prManager: IPullRequestManager, reviewManager: ReviewManager) {
+export function registerCommands(context: vscode.ExtensionContext, prManager: IPullRequestManager,
+	reviewManager: ReviewManager, telemetry: ITelemetry) {
 	// initialize resources
```

Comment:
should we also log an event on these commands? openPullRequestInGitHub, deleteLocalBranch, approve, requestChanges, openDescription?